specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-04-26/customer-insights.json

```diff
@@ -3149,6 +3149,378 @@
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "predictionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Prediction.",
+            "pattern": "^[a-zA-Z][a-zA-Z0-9_]+$",
+            "minLength": 1,
+            "maxLength": 512
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PredictionResourceFormat"
+            },
+            "description": "Parameters supplied to the create/update Prediction operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully created the Prediction.",
+            "schema": {
+              "$ref": "#/definitions/PredictionResourceFormat"
+            }
+          },
+          "202": {
+            "description": "Accepted. The create Prediction operation is accepted."
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "Predictions"
+        ],
+        "operationId": "Predictions_Get",
+        "x-ms-examples": {
+          "Predictions_Get": {
+            "$ref": "./examples/PredictionsGet.json"
+          }
+        },
+        "description": "Gets a Prediction in the hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "predictionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Prediction."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully get the Prediction.",
+            "schema": {
+              "$ref": "#/definitions/PredictionResourceFormat"
```

Comment:
it's a good suggestion I think. But we may need to do it for all of our resources in order to keep consistency, which will be breaking change and we don't want at this point.